test/_init_tests.js

```diff
@@ -91,13 +91,14 @@ afterEach(() => {
   window.ENABLE_LOG = false;
   window.AMP_DEV_MODE = false;
   window.context = undefined;
+  if (sandboxes.length > 0) {
+    sandboxes.splice(0, sandboxes.length).forEach(sb => sb.restore());
```

Comment:
So this actually restores all sandboxes that was initiated, why are we still throwing an error below?